File: api/Models/Entidades/Usuario.cs
using Dapper.Contrib.Extensions;

namespace Models.Entidades;

[Table("Usuario")]
public class Usuario
{
    [Key]
    public int UsuarioId { get; set; }
    public string Login { get; set; } = string.Empty;
    public string Senha { get; set; } = string.Empty;
    public string UsuarioNome { get; set; } = string.Empty;
    public DateTime CriadoEm { get; set; }
}

File: catalogo-personagens/src/app/shared/models/entidades/usuario.ts
export interface Usuario {
  usuarioId: number;
  login: string;
  senha: string;
  usuarioNome: string;
  criadoEm: Date;
}

export interface UsuarioLogin {
  usuario: string;
  senha: string;
}

export interface UsuarioLogado {
  token: string;
}
